docker/assets/entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+/app/pat --config /app/config.json \
```

Comment:
I would prefer that we avoid adding this file.

* We can (to some degree) customize the paths by setting the standard XDG variables `$XDG_DATA_HOME`, `$XDG_CONFIG_HOME` and `$XDG_STATE_HOME`.
* Overriding the callsign with `--mycall` prevents Pat from reading the Winlink password from config, so that should also be avoided IMO. It's better to set mycall in config.

The remaining `pat http` could be set directly in Dockerfile.

Reply:
> I would prefer that we avoid adding this file.
> 
>     * We can (to some degree) customize the paths by setting the standard XDG variables `$XDG_DATA_HOME`, `$XDG_CONFIG_HOME` and `$XDG_STATE_HOME`.
> 
>     * Overriding the callsign with `--mycall` prevents Pat from reading the Winlink password from config, so that should also be avoided IMO. It's better to set mycall in config.
> 
> 
> The remaining `pat http` could be set directly in Dockerfile.

I'll try to figure out if the `XDG_*` vars work, but it seems like Pat won't run in the container if I don't set `--mycall`. When I set both `--mycall` and configure a callsign in `config.json` it takes the value in `config.json` regardless of the argument set by `--mycall`. I don't know if that's expected.